staging/src/k8s.io/controller-manager/pkg/leadermigration/migrator_test.go

```diff
@@ -106,6 +122,28 @@ func TestLeaderMigratorFilterFunc(t *testing.T) {
 				"cloud-node-lifecycle": ControllerMigrated,
 			},
 		},
+		{
+			name:      "wildcard config, kcm",
+			config:    wildcardConfig,
+			component: "kube-controller-manager",
+			expectResult: map[string]FilterResult{
+				"deployment":           ControllerNonMigrated,
+				"route":                ControllerMigrated,
+				"service":              ControllerMigrated,
+				"cloud-node-lifecycle": ControllerMigrated,
```

Comment:
Can we add a test case for "cloud-node" as its a CCM ControllerNonMigrated?

Reply:
Sure. Adding it should not affect the tests in anyway, but it is good for clarity